elements/bulbs-page/bulbs-page.js

```diff
@@ -28,6 +29,35 @@ export default class BulbsPage extends BulbsHTMLElement {
     InViewMonitor.remove(this);
   }
 
+  get dimensions () {
```

Comment:
Agree with @brycedorn here, the two of you should work together on bubbling this up to `util` so it can be used in all 4 places: 

- `bulbs-page`
- `reading-list-article`
- Slick Slideshow
- Clickventure